### PR TITLE
Nwz/localization choice

### PR DESCRIPTION
**Pull request by nicolaswurtz**

### CODE REVIEW

File: front/src/common/ChangeLanguage.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        >
+          {formatLanguageWithFlag(lng.key)}
+          {lng.value}
+        </button>
+      ))}
+    </>
+  );
+};
+
+export default function ChangeLanguage() {
```
Comment (clarani) [inline front/src/common/ChangeLanguage.tsx]:
```suggestion
export default function ChangeLanguageModal() {
```

File: front/src/common/BootstrapSNCF/NavBarSNCF.jsx [hunk 2]
```diff
@@ -72,13 +62,24 @@ export default function LegacyNavBarSNCF(props) {
                 <span className="mr-2">
                   <FaInfoCircle />
                 </span>
-                {t('NavBar.about')}
+                {t('about')}
+              </button>,
+              <button
+                type="button"
+                className="btn-link text-reset"
+                onClick={() => openModal(<ChangeLanguage />, 'sm')}
```
Comment (clarani) [inline front/src/common/BootstrapSNCF/NavBarSNCF.jsx]:
```suggestion
                onClick={() => openModal(<ChangeLanguageModal />, 'sm')}
```

File: front/src/reducers/main.ts [hunk 3]
```diff
@@ -5,6 +5,17 @@ import { ThunkAction, Notification } from '../types';
...
+export const UPDATE_LANGUAGE = 'main/UPDATE_LANGUAGE';
+type ActionUpdateLangage = { type: typeof UPDATE_LANGUAGE };
+export function updateLangage(language: string): ThunkAction<ActionUpdateLangage> {
+  return (dispatch) => {
+    dispatch({
+      type: UPDATE_LANGUAGE,
+      language,
+    });
+  };
+}
```
Comment (clarani) [inline front/src/reducers/main.ts]:
```suggestion
```
This is not used, so we should not commit this (dead code), same for all the changes of this file

**Review by clarani — CHANGES_REQUESTED** [review]
Thanks for this PR :)

File: front/src/common/ChangeLanguage.tsx (hunk 1 above)
Comment (nicolaswurtz) [reply front/src/common/ChangeLanguage.tsx]: You're right !

File: front/src/reducers/main.ts (hunk 3 above)
Comment (nicolaswurtz) [reply front/src/reducers/main.ts]: Oh, it was a first try, never used finally (i18n has it's own localstorage variable!). Thanx !

**Review by clarani — DISMISSED** [review]
Ok, LGTM ! I have tested in local, no issue on my side :)

**Review by clarani — APPROVED** [review]

**Pull request merged**